src/ui/Quote/Testimonial/Testimonial.jsx

```diff
@@ -12,9 +11,9 @@ Testimonial.propTypes = {
 function Testimonial({ children, ...rest }) {
   return (
     <div className="eea testimonial" {...rest}>
-      <Divider />
+      <Divider className="green clean" />
```

Comment:
@g-stamatis Can we use semantic ui `fitted` option instead of custom `clean` class?

https://react.semantic-ui.com/elements/divider/#variations-fitted

Reply:
@g-stamatis Can we align the `color` class names with what we have for cards?
* `primary` - blue
* `secondary` - green
* `tertiary` - whatever color is that :)
* (default) - no value - gray

Reply:
@g-stamatis At some point, we may want some text or other components within the divider. Can you make sure it's properly styled, at least with text within it?

See https://react.semantic-ui.com/elements/divider/#types-horizontal-table